packages/ragbits-core/src/ragbits/core/vector_store/chromadb_store.py

```diff
@@ -10,20 +10,29 @@
     HAS_CHROMADB = False
 
 from ragbits.core.embeddings import Embeddings
+from ragbits.core.metadata_store.base import MetadataStore
 from ragbits.core.utils.config_handling import get_cls_from_config
 from ragbits.core.vector_store import VectorDBEntry, VectorStore
 
 
 class ChromaDBStore(VectorStore):
     """Class that stores text embeddings using [Chroma](https://docs.trychroma.com/)"""
 
+    CHROMA_IDS_KEY = "ids"
```

Comment:
tbh I'm not a fan of extracting all constants. It makes the code harder to read later on. Additionally, it clutters the ChromaDBStore public interface; users of this class will receive autosuggestions with a lot of unnecessary variables after typing `ChromaDBStore.`